### DISCUSSION TITLE
Link media files in link fieldtype

### DISCUSSION THREAD

**Discussion by laborb-mp** [statamic/cms#6003] [Q&A]
Hey there!

I'd like to know if it is possible to link media files (e.g. a `*.pdf` File or an image etc.) within a link fieldtype (https://statamic.dev/fieldtypes/link) and within a bard field button (https://statamic.dev/fieldtypes/bard)?

If not: what would be the best approach to achieve this behavior?
Extending or creating a new fieldtype? 

Thanks in advance!

**Comment by edalzell** [comment]
If you don't want to build anything your self (or PR this functionality into Statamic's core), you can use [LinkIt](https://statamic.com/addons/rias/link-it)

**Comment by jasonvarga** [comment]
You can select an asset container when configuring the link fieldtype. This will allow you to link to assets. Images, PDFs, whatever.
